src/main/java/edu/cornell/kfs/module/purap/batch/service/JaggaerGenerateContractPartyCsvService.java

```diff
@@ -1,13 +1,14 @@
 package edu.cornell.kfs.module.purap.batch.service;
 
+import java.sql.Date;
 import java.util.List;
 
 import edu.cornell.kfs.module.purap.CUPurapConstants.JaggaerContractUploadProcessingMode;
 import edu.cornell.kfs.module.purap.businessobject.lookup.JaggaerContractUploadBaseDto;
 
 public interface JaggaerGenerateContractPartyCsvService {
     
-    public List<JaggaerContractUploadBaseDto> getJaggerContractsDto(JaggaerContractUploadProcessingMode processingMode, String processingDate);
+    public List<JaggaerContractUploadBaseDto> getJaggerContractsDto(JaggaerContractUploadProcessingMode processingMode, Date processingDate);
```

Comment:
Since you're already making other changes to this method, I would suggest fixing the typo in its name as well: "Jagger" --> "Jaggaer"

Reply:
ok